troi/playlist.py

```diff
@@ -209,6 +212,51 @@ def submit(self, token, created_for=None, algorithm_metadata=None):
 
         return playlist_mbids
 
+    def _lookup_spotify_ids(self, recordings):
+        """ Given a list of Recording elements, try to find spotify track ids from labs api spotify lookup using mbids
+        and add those to the recordings. """
+        response = requests.post(
+            SPOTIFY_IDS_LOOKUP_URL,
+            json=[{"[recording_mbid]": recording.mbid} for recording in recordings]
+        )
+        response.raise_for_status()
+        spotify_data = response.json()
+        for recording, lookup in zip(recordings, spotify_data):
+            recording.spotify_id = lookup["spotify_track_id"]
+
+    def submit_to_spotify(self, user_id: str, token: str, is_public: bool = True, is_collaborative: bool = False):
```

Comment:
On a tangent, I had used a TypedDict for params earlier but dropped it because according to setup.py troi support Python 3.6. I am not sure whether troi would actually run with Python 3.6 (even 3.7 or 3.8) at the point because I don't think we just tested with whatever version LB runs on which is usually newer. We should decide which versions we want to support and according update the CI to ensure that troi indeed runs with those versions.

Reply:
Discussed to settle on Python 3.8